packages-ts/starknet-gauntlet-ledger/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@chainlink/starknet-gauntlet-ledger",
+    "version": "0.0.1",
+    "description": "Gauntlet Starknet Ledger Signer",
+    "keywords": [
+      "typescript",
+      "ledger",
+      "cli"
+    ],
+    "main": "./dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+      "dist/**/*",
+      "!dist/**/*.test.js"
+    ],
+    "scripts": {
+      "lint": "tsc",
+      "test": "SKIP_PROMPTS=true jest --runInBand --config ../../jest.config.ts --selectProjects starknet-gauntlet-starkgate --passWithNoTests",
+      "test:ci": "yarn test --ci",
+      "format": "yarn prettier --write ./src",
+      "format:check": "yarn prettier --check ./src",
+      "clean": "rm -rf ./dist/ ./bin/",
+      "build": "yarn clean && tsc -b",
+      "bundle": "yarn build && pkg ."
+    },
+    "dependencies": {
+      "@chainlink/starknet-gauntlet": "*",
+      "@ledgerhq/hw-app-starknet": "^1.0.1",
+      "@ledgerhq/hw-transport-node-hid": "^6.27.2",
+      "starknet": "^3.15.0"
```

Comment:
This seems to re-import `starknet.js`, Kristijan recently moved this to the toplevel package.json: https://github.com/smartcontractkit/chainlink-starknet/blob/193c1e40be99a0016391b2cff13f53299bdc8e4c/package.json#L59

That breaks using the package as a library outside of the workspace though. I think we should move the import to`starknet-gauntlet`, then re-export it there